cognite/seismic/protos/query_service_messages.proto

```diff
@@ -110,18 +101,6 @@ message GeometryCubeRequest {
     bool include_trace_header = 3;
 }
 
-/**
-Request a time slice from a file and filter by range of inlines and crosslines
-**/
-message LineTimeSliceQueryRequest {
-    Identifier file = 1;
-    LineBasedRectangle rectangle = 2;
-    oneof z {
-        int32 constant = 3;
-        Horizon horizon = 4;
```

Comment:
Would horizons be removed with these changes?

Reply:
I was just thinking about it, I will put it back on.

Reply:
At first I though it's easily replacable with getVolume, but there's horizon thingy

Reply:
I'm wondering if it would make sense to have the min->max / constant / horizon choice as a `oneof` in `Range3D`. I think currently the `LineTimeSliceQueryRequest` has some custom code for constant/horizon indices but maybe that could be moved into the trace stores

Reply:
I'm not super familiar with `Horizon` but I think it's a 2d array which defines the surface(pls correct me if I'm wrong), so in that case it doesn't define a range but rather a surface, so I think it shouldn't be part of `Range3D`

Reply:
Regarding moving custom horizon code to trace stores, I think it can be done, tho I think that's the only place we use horizons so far, so it might not be super important

Reply:
The `Horizon` is just a 1d array which needs to be as long as the number of traces being returned I believe. But yeah, I think you're right, it's not defining a range, and moving that much code for the v0 api seems not that important